extensions/amp-addthis/amp-addthis.md

```diff
@@ -0,0 +1,95 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-addthis"></a> `amp-addthis`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Displays an <a href="https://www.addthis.com">AddThis</a> website tools embed.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Experimental</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-addthis" src="https://cdn.ampproject.org/v0/amp-addthis-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>fill, fixed, fixed-height, flex-item, nodisplay, responsive</td>
+  </tr>
+</table>
+
+## Behavior
+
+The `amp-addthis` component provides the ability to embed AddThis website tools (for example, 
+sharing buttons, follow buttons, related posts, etc.) on an AMP page.
+
+Example:
+```html
+<amp-addthis
+  width="320"
+  height="92"
+  data-pub-id="ra-59c2c366435ef478"
+  data-widget-id="0fyg">
+</amp-addthis>
+```
+
+## Attributes
+
+**data-pub-id**
+
+The AddThis publisher ID found in the URL in the [AddThis dashboard](https://addthis.com/dashboard)
+after logging in. For example, in the URL `https://www.addthis.com/dashboard#gallery/pub/ra-55e761259a1acdc2`,
+`ra-55e761259a1acdc2` is the publisher ID.
+
+**data-widget-id**
+
+The AddThis widget ID for the tool to be displayed, also found on the [AddThis dashboard](https://addthis.com/dashboard).
+The widget Id for a specific tool can be found by opening that tool in the AddThis dashboard and
+copying the last part of the URL. For example, in the URL `https://www.addthis.com/dashboard#tool-config/pub/ra-55e761259a1acdc2/widgetId/xz1d`,
+`xz1d` is the widget Id.
+
+**data-share-title**
+
+Optional. If set, this is the title that the AddThis tool will attempt to share when sharing occurs.
+If not set, the title of the document containing the amp-addthis tag will be used.
+
+**data-share-url**
+
+Optional. If set, this is the URL that the AddThis tool will attempt to share when sharing occurs.
+If not set, the `location.href` property of the document containing the amp-addthis tag will be
+used.
+
+**data-share-media**
+
+Optional. If set, this is an URL for a piece of media (e.g., image or video) that the AddThis tool
+will attempt to share when sharing occurs. If not set, this is left undefined.
+
+**data-share-description**
+
+Optional. If set, this is the description of the page that the AddThis tool will attempt to share
+when sharing occurs. If not set, this is left undefined.
+
+## Validation
+
+See amp-addthis rules in the AMP validator specification.
+
+## Privacy
```

Comment:
@dmvjs What this component collects needs from the page and user (e.g. clicks, scrolls, etc) needs to be documented as well.